assets/app/view/part/town_rect.rb

```diff
@@ -136,7 +197,12 @@ def preferred_render_locations
         end
       end
 
-      def render
+      def load_from_tile
+        @edge = @tile.preferred_city_town_edges[@town]
+        @towns = @tile.towns.size
```

Comment:
num_towns or whatever you call the city ones

Reply:
9488efb